### DISCUSSION TITLE
Send notification to device to include "Front Door Last Operator

### DISCUSSION THREAD

**Discussion by intrinsic777** [jbergler/hass-ttlock#127] [Q&A]
I have set up a notification to send a message to my Wife's phone when the door is successfully unlocked. The content of the message is just "door unlocked". I want to be able to send a message based on the "sensor.front_door_last_operator" that includes the name of the person that unlocked the door. HA shows this detail, I just want to include it in a message notification if possible.
My current code is below.

alias: Front door unlocked to Vicky's phone.
description: ""
trigger:
  - platform: device
    device_id: 020d548f7d7c8c66b88b55341f863c55
    domain: lock
    entity_id: f07ff9bde557726bdeb95fd82de2d80f
    type: unlocked
condition: []
action:
  - device_id: f01c0329633e8bb0e95ed18f0655197d
    domain: mobile_app
    type: notify
    message: Front Door Unlocked
    data:
      ttl: 0
      priority: high
      car_ui: true
      channel: ha_notifications
      importance: high

**Comment by jbergler** [comment]
Hiya, are you familiar with how to use templating logic in home assistant? The documentation is here: https://www.home-assistant.io/docs/configuration/templating/

Something like this might get you started.

```yaml
message: "Front Door Unlocked by {{states('sensor.front_door_last_operator')}}"
```

**Comment by intrinsic777** [comment]
Many thanks. I have looked at the documentation and fiddled within my limited capability.  As a start I used your suggestion but that only pulls back the previous operator, not the current.  For instance if my wife unlocked the door 2 hours ago and I now unlocked he door, I would get the message that my wife unlocked it, not me. It seems to pull back the preiously stored operator, not the live one. Trying hard to solve this myself but really struggling!
